hack/gc-host-local.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+function echoerr {
+    >&2 echo "$@"
+}
+
+CLEAN=false
+DATA_DIR="/run/antrea/cni/networks/antrea/"
+BACKUP_DIR="${DATA_DIR}leaked/"
```

Comment:
```suggestion
DATA_DIR="/run/antrea/cni/networks/antrea"
BACKUP_DIR="${DATA_DIR}/leaked/"
```

Reply:
Fixed. I think we should also remove the last "/" from BACKUP_DIR